angularjs-1.5-basics/src/app/interval-example/interval-example.controller.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var intervalExampleController = function ($interval, $log) {
+    var intervalPromise;
+    var vm = this;
+
+
+    vm.count = 0;
+    vm.startIncrementTimer = function (timeInMinutes) {
+        intervalPromise = $interval(function () {
+            $log.debug('incremented count by 1, current count is - ' + vm.count);
+            vm.count += 1;
+        }, 1000, timeInMinutes * 60 * 1000);
+    };
+
+    vm.stopIncrementTimer = function () {
+        vm.count = 0;
+        $interval.cancel(intervalPromise);
+        $log.debug('reset counter to 0, current count is' + vm.count);
+    };
+
+};
+
+
+angular.module('interval-example').controller('IntervalExampleController', intervalExampleController);
```

Comment:
__[jshint]__
* 'angular' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->